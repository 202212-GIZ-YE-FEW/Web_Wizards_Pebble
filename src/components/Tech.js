const Tech = () => {
    return (
        <section id='about' className='pt-20 lg:pt-[120px] pb-10 lg:pb-20'>
            <div className='container'>
                <div className='flex flex-wrap -mx-4'>
                    <div className='w-full px-4'>
                        <div className='text-center mx-auto mb-[60px] max-w-[620px]'>
                            <h2
                                className='
                  font-bold
                  text-3xl
                  sm:text-4xl
                  md:text-[42px]
                  text-dark
                  mb-4
                '
                            >
                                Technologies Used and Project Structure
                            </h2>
                            <p
                                className='
                  text-lg sm:text-xl leading-relaxed sm:leading-relaxed text-body-color
                '
                            >
                                Body placeholder for text paragraph. A paragraph
                                is a self-contained unit of text dealing with a
                                particular point or idea.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </section>
    );
};

export default Tech;
