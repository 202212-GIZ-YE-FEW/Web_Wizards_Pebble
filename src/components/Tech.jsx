import React from "react";

const Tech = (props) => {
    const { t } = props;

    return (
        <section id='tech' className='m-16 md:m-36'>
            <h2 className='mb-2 text-center text-2xl font-bold md:mb-4 md:text-4xl lg:mb-4 lg:text-4xl'>
                {t("techTitle")}
            </h2>
            <p className='https://us05web.zoom.us/j/83164148611?pwd=M3c2M1V0U1U5aVFabTVjTzh1b3pidz09'>
                {t("techBrief")}
            </p>
        </section>
    );
};

export default Tech;
