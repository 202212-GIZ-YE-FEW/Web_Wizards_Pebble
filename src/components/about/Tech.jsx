import Image from "next/image";

const Tech = (props) => {
    const { t } = props;

    return (
        <div className='h-screen flex flex-col justify-center'>
            <div className='max-w-screen-xl mx-auto px-4 sm:px-6 lg:px-8'>
                <h2 className='mb-4 text-center text-2xl md:text-4xl font-bold  lg:text-5xl lg:font-bold m-1 md:m-4 font-sans'>
                    {t("techTitle")}
                </h2>
                <div className='flex flex-wrap justify-center'>
                    {techs.map((tech) => (
                        <div key={tech.tech} className='m-6'>
                            <Image
                                src={`/images/tech/${tech.logo}`}
                                alt={tech.tech}
                                width={100}
                                height={100}
                                className='max-h-12'
                            />
                        </div>
                    ))}
                </div>
            </div>
        </div>
    );
};

export default Tech;
