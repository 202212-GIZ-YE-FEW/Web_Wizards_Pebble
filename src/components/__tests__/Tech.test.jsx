import renderer from "react-test-renderer";

import Tech from "../about/Tech";

it("renders correctly", () => {
    const tree = renderer.create(<Tech t={(str) => str} />).toJSON();
    expect(tree).toMatchSnapshot();
});
