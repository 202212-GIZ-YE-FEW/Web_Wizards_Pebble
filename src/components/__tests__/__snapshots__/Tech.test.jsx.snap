// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="m-16 md:m-36"
  id="tech"
>
  <h5
    className="text-center mb-4"
  >
    techStack
  </h5>
  <h2
    className="mb-2 text-center font-sans text-2xl font-bold md:mb-4 md:text-4xl lg:mb-4 lg:text-4xl"
  >
    techTitle
  </h2>
  <p
    className="text-center"
  >
    React.js/Next.js for coding, TailwindCSS for styyling and Jest for testing
  </p>
</section>
`;
